metagraph/src/cli/config/config.hpp

```diff
@@ -91,6 +91,7 @@ class Config {
     unsigned int genome_binsize_anno = 1000;
     unsigned int arity_brwt = 2;
     unsigned int relax_arity_brwt = 10;
+    unsigned long long disk_buffer_size = 10'000; // in B
```

Comment:
Is this the buffer for an int_vector_buffer? They say modern SSDs' page size is 8192 bytes. So maybe set this to a multiple of that, like 16k? Also, what about hardcode this and not expose as a parameter? I guess we aren't going to change this ever, are we?

Reply:
Yes, this is for `int_vector_buffer`. I exposed this as a parameter because of the kinda unexpected behaviour (in terms of performance) depending on this value. I thought it may be convenient to leave this as a CLI parameter to be able to easily verify if changing this may speed up execution time.
Let me know if you want me to change it to the hardcoded. A lower number of CLI parameters is less confusing for the end-user.

Reply:
Ok, then let's figure out a name for this parameter.

This is the size of the buffer in int_vector_buffer. I guess it makes sense to use the same value for all int_vector_buffer instances that are used in the same way, that is, doing random access on them.
What about `RA_ivbuffer_size`?

Reply:
Ok, I will rename it into `RA_ivbuffer_size` and make it an "advanced" parameter.
As I see `int_vector_buffer` is used in many places in the code. I guess most or all except row_disk annotations are not random access. Am I right? Are you aware of any random access use cases beyond `row_sparse_disk`?